frontend/src/components/navbar/Navbar.scss

```diff
@@ -1,9 +1,104 @@
+@import "../../constants.scss";
+
 nav.navbar {
-  height: 4em;
-  background-color: #bbffff;
+  height: 2em;
+  background-color: $secondary;
+  color: $text-color-light;
   padding: 0.5em;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+
+  ul {
+    padding: 0;
+    margin: 0;
+    width: 100%;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  a:visited {
+    color: $text-color-light;
+  }
+
+  a:hover {
+    text-decoration: underline;
+  }
 
   img.navbar__logo {
+    height: 2em;
+  }
+
+  h1 {
+    font-size: medium;
+  }
+
+  .menu-items {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 10px;
+
+    .admin-links {
+      margin-left: auto;
+    }
+
+    .admin-only-links {
+      display: flex;
+      flex-direction: row;
+      justify-content: flex-end;
+      align-items: center;
+      gap: 10px;
+      text-decoration: none;
+    }
+  }
+
+  @media only screen and (max-width: 650px) {
+    .menu-items {
+      display: none;
+    }
+
+    .mobile-menu {
+      display: block;
+      position: fixed;
+      left: 0;
+      z-index: 2;
+    }
+  }
+
+  @media only screen and (min-width: 651px) {
+    .mobile-menu {
+      display: none;
+    }
+  }
+
+  .menu-items.expanded {
     height: 100%;
+    width: 70%;
+    background-color: $secondary;
+    z-index: 1;
+    position: fixed;
+    left: 0;
+    top: 0;
+    padding-top: 35px;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+
+    .admin-links {
+      margin-left: 0;
+    }
+
+    .admin-only-links {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-around;
+      margin-left: 0;
+    }
```

Comment:
What's the difference between `.admin-links` and `.admin-only-links`? I think we may need a more descriptive name!